site/content/blog/markdowndb-launch.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Announcing the official launch of MarkdownDB! 🚀
+description: We're excited to announce the official launch of MarkdownDB, an open library to transform markdown content into SQLite database. And now, it has a dedicated website at markdowndb.com
```

Comment:
"We're excited" is boilerplate 😄 Let's trim. "official" is boilerplate etc. We also can reuse what we have in the hero. So how about:

Announcing MarkdownDB, an open source library to transform markdown content into sql-queryable data. Get a rich SQL API to your markdown files in seconds. Build rich markdown-powered sites easily and reliably. New dedicated website at markdowndb.com

Reply:
Agree, just removing this part - Get a rich SQL API to your markdown files in seconds. - as it's already stated in the title.